### ISSUE TITLE
Content labeling

### ISSUE THREAD

**Issue by mtotschnig** [mtotschnig/MyExpenses#547]
Play Store pre-launch report identifies 45 issues with missing content labels:

1. org​.totschnig​.myexpenses:id​/AmountEditText: This must be handled in Compound view class AmountInput (https://developer.android.com/guide/topics/ui/accessibility/custom-views)
